src/common/less/constants.less

```diff
@@ -1,7 +1,7 @@
 /* ********** Variables ********** */
 @owDesktopMaxWidth: 1200px;
 @owTabletBreakpoint: 1024px;
-@owMobileBreakpoint: 768px;
+@owMobileBreakpoint: 767px;
```

Comment:
Have you checked that this doesn't leave a 1 px fucked-up css state for our other pages?

Reply:
watch your language young man
but yeah, is this gonna leave some 👉👌🆙 CSS @oleast?